packages/react-vapor/src/components/textarea/TextArea.tsx

```diff
@@ -31,6 +28,9 @@ export interface ITextAreaOwnProps {
     isAutosize?: boolean;
 
     onChangeCallback?: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
+    validate?: (value: string) => boolean;
+    validationMessage?: string;
+    validationLabelProps?: ILabelProps;

```

Comment:
Since the `validationMessage` goes into a hardCoded `<label />` and you can pass props to this label with `validationLabelProps`. I wonder if we could not simply pass a ReactNode to a `invalidLabel` Props.